website/content/docs/get-started/developer-qs.mdx

```diff
@@ -167,6 +169,66 @@ import org.springframework.vault.core.VaultTemplate;
 
 </CodeBlockConfig>
 
+</Tab>
+<Tab heading="OpenAPI Go (Beta)" group="openAPI-go">
+
+
+[OpenAPI Go](https://github.com/hashicorp/vault-client-go) (Beta) client library:
+
+```shell-session
+$ go get github.com/hashicorp/vault-client-go
+```
+
+Now, let's add the import statements for the client library to the top of the file.
+
+<CodeBlockConfig heading="import statements for client library" lineNumbers>
+
+```go
+import (
+	"github.com/hashicorp/vault-client-go"
+	"github.com/hashicorp/vault-client-go/schema"
+)
+```
+
+</CodeBlockConfig>
+
+
+</Tab>
+<Tab heading="OpenAPI C# (Beta)" group="openAPI-cs">
+
+
+[OpenAPI C#](https://github.com/hashicorp/vault-client-dotnet) (Beta) client library:
+
+Vault is a package available at [Hashicorp Nuget](https://www.nuget.org/profiles/hashicorp).
+
+
+```shell-session
+ nuget install HashiCorp.Vault -Version "0.1.0-beta"
+    -name HashicorpArtifactory \ 
+    -source https://artifactory.hashicorp.engineering/ui/repos/tree/General/vault-devex-nuget-local \
+    -username "myusername" \
+    -password "mypassword"
+```
+
+**Or:**
+
+```shell-session
+dotnet add package Hashicorp.Vault -version "0.1.0-beta" 
```

Comment:
Sorry. I'm not great at the suggestion features, but the flags below this line can now be deleted, e.g. 
`    --name HashicorpArtifactory \
    --username "myusername" \
    --password "mypassword"`
can be removed